services/ticker/internal/gql/main.go

```diff
@@ -117,3 +111,13 @@ func (r *resolver) Serve(address string) {
 		r.logger.Errorln("server.ListenAndServe:", err)
 	}
 }
+
+// SetupHandler sets up the response handler.
+func (r *resolver) SetupHandler() relay.Handler {
```

Comment:
Nice separation to improve the tests 👏 🎉 .

Suggest renaming this to indicate it is creating a new relay handler. SetupHandler sounds like the internals of the resolver are being setup, but actually this just creates a new handler from the resolver.
```suggestion
func (r *resolver) NewRelayHandler() relay.Handler {
```